src/Response.php

```diff
@@ -159,9 +156,8 @@ public function getReasonPhrase()
      */
     public function withStatus($code, $reasonPhrase = '')
     {
-        $this->validateStatus($code);
```

Comment:
Ok, but why? We need to repeat this code hear and in contructor, so private setter keep setting status code always correct